index.html

```diff
@@ -438,22 +438,22 @@ <h6 class=author-name>Steven King</h6>
         <div class="row">
             <div class="col-4 col-md-6 col-sm-8 col-xs-12">
```

Comment:
439 row - try to use col-md-8 instead 6 ant try responsive design. It worked for me.